docs/operating-system/debugging.md

```diff
@@ -9,21 +9,59 @@ This section is not for end users. End users should use the [SSH add-on] to SSH
 
 [SSH add-on]: https://github.com/home-assistant/hassio-addons/tree/master/ssh
 
-## SSH access to the host
+## Enabling SSH access to the host
 
 :::info
 SSH access through the [SSH add-on] (which will give you SSH access through port 22) will not provide you with all the necessary privileges, and you will be asked for a username and password when typing the 'login' command. You need to follow the steps below, which will setup a separate SSH access through port 22222 with all necessary privileges.
 :::
 
-### Home Assistant Operating System
+### USB CONFIG Access Method
 
 Use a USB drive formatted with FAT, ext4, or NTFS and name it CONFIG (case sensitive). Create an `authorized_keys` file (no extension) containing your public key, and place it in the root of the USB drive. File needs to be ANSI encoded (not UTF-8) and must have Unix line ends (LF), not Windows (CR LF). See [Generating SSH Keys](#generating-ssh-keys) section below if you need help generating keys. From the UI, navigate to the Supervisor system page and choose "Import from USB". You can now access your device as root over SSH on port 22222. Alternatively, the file will be imported from the USB when the Home Assistant OS device is rebooted.
 
 :::tip
 Make sure when you are copying the public key to the root of the USB drive that you rename the file correctly to `authorized_keys` with no `.pub` file extension.
 :::
 
-You should then be able to SSH into your Home Assistant device. On Mac/Linux, use:
+
+### SSH Plugin Access Method
```

Comment:
This requires a third-party / community add-on. This should not be part of the official documentation.

Reply:
Got it.